src/colornames.csv

```diff
@@ -13702,6 +13735,7 @@ Jodhpur Tan,#dad1c8,
 Jodhpurs,#ebdcb6,
 Jogging Path,#c0b9a9,
 John Lemon,#eeff22,x
+John's Blue,#89a5c2,

```

Comment:
Thats sounds like someone working there called John that just likes blue. Lets rename or remove that one